lib/bug_report_templates/active_admin_master.rb

```diff
@@ -8,22 +8,23 @@
 gemfile(true) do
   source 'https://rubygems.org'
 
-  gem 'rails', require: false
-  gem 'sqlite3', platform: :mri
+  # Use local changes or ActiveAdmin master.
+  if ENV['ACTIVE_ADMIN_PATH']
+    gem 'activeadmin', path: ENV['ACTIVE_ADMIN_PATH'], require: false
```

Comment:
did you know this
`bundle config local.rack ~/Work/git/rack`
http://bundler.io/v1.5/man/bundle-config.1.html#LOCAL-GIT-REPOS